src/core/geometry/poly3/toString.js

```diff
@@ -1,11 +1,12 @@
-const plane = require('')
-const vert3 = require('')
+const plane = require('../../math/plane/')
+const vec3 = require('../../math/vec3/')
 
-const toString = poly3 => {
-  let result = 'Polygon3 plane: ' + plane.toString(poly3.plane) + '\n'
+const toString = (poly3) => {
+  let result = 'poly3: plane: ' + plane.toString(poly3.plane) + ' vertices: ['
```

Comment:
hey, you can call es6 template strings to the rescue ! :)

Reply:
ie this can be simplified to 
```javascript
const result = `poly3: plane: ${plane.toString(poly3.plane)},   vertices: [${poly3.vertices.map(v => vec3.toString(v)]`